spec/graphql/schema/object_spec.rb

```diff
@@ -375,6 +375,7 @@ def self.type_error(err, ctx)
         shape.delete(:@configs)
         shape.delete(:@future_schema)
         shape.delete(:@metadata)
+        shape.delete(:@comment)
```

Comment:
I think we should include `@comment` in our shape tests so that we can be sure that GraphQL-Ruby is creating efficient object shapes by default. 

We could remove this, and instead, add a line here: 

https://github.com/rmosolgo/graphql-ruby/blob/a794fc8e88660fa7d58837c25797cf529e4d45f0/lib/graphql/schema/member/base_dsl_methods.rb#L56-L59

For example, 

```ruby
child_class.comment(comment)
```

Which would initialize the instance variable and also implement comment inheritance. Alternatively, if you don't want inheritance, you could do `child_class.comment = NOT_CONFIGURED` (we'd need an `attr_writer` in that case)

Reply:
Thanks, I've added the "cloning" of the comment to the child, seemed like a better option then setting it to empty default.